proxy/src/main/java/net/md_5/bungee/BungeeCord.java

```diff
@@ -462,7 +462,14 @@ public String getTranslation(String name, Object... args)
     @Override
     public int getOnlineCount()
     {
-        return connections.size();
```

Comment:
This method is designed not to lock on purpose. If you want a locked count use getPlayers().size()